mw-std/src/loader.rs

```diff
@@ -58,6 +58,7 @@ pub struct Instance {
 }
 
 impl Instance {
+    #[allow(dead_code)]
```

Comment:
You can `pub` this interface. Others can't use this method?

Reply:
maybe i need coffe to wake up